metapackages/auto-instrumentations-node/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@opentelemetry/auto-instrumentations-node",
-  "version": "0.39.1",
+  "version": "0.39.2",
```

Comment:
Since instrumentation introduces a minor version bump, I am expecting the auto-instrumentation package to bump a minor version too, so that end users depending on `^0.39.1` will net get breaking changes if they depend on caret range.

@dyladan do you know if release please supports this?

Reply:
documented in #1659